docker/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM amd64/ubuntu:22.04
 ARG DEBIAN_FRONTEND=noninteractive
 ENV TZ=UTC
 RUN apt-get update && \
-    apt-get install -y wget openjdk-17-jre && \
```

Comment:
Let's keep jdk 17 in our docker image. The docker env is ours! We can use any version we want ;-)